src/explore-education-statistics-admin/src/pages/publication/components/PublicationDraftReleases.tsx

```diff
@@ -28,13 +34,16 @@ const PublicationDraftReleases = ({ releases, onChange }: Props) => {
   const [showDraftStatusGuidance, toggleDraftStatusGuidance] = useToggle(false);
   const [showIssuesGuidance, toggleIssuesGuidance] = useToggle(false);
 
+  if (releases.length === 0) {
+    return <InsetText>There are no draft releases.</InsetText>;
+  }
+
```

Comment:
The spacing is a bit wrong when there are no draft releases.
<img width="533" alt="spacing" src="https://user-images.githubusercontent.com/81572860/187661119-17e6da3f-6008-4b63-9e98-88bad4cddbe0.PNG">